specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2022-10-01-preview/common-types.json

```diff
@@ -0,0 +1,71 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-10-01",
+    "title": "Cost Management common types"
+  },
+  "paths": {},
+  "definitions": {
+    "ErrorResponse": {
```

Comment:
>ErrorResponse [](http://example.com/codeflow?start=5&length=13)

Consider just leveraging the definitions from common-types instead:

[https://github.com/Azure/azure-rest-api-specs-pr/blob/main/specification/common-types/resource-management/v3/types.json#L336](https://github.com/Azure/azure-rest-api-specs-pr/blob/main/specification/common-types/resource-management/v3/types.json#L336)